### PR TITLE
Update todo for latest widgets/stores and cli commands

### PR DESCRIPTION
**Pull request by agubler**
Update `todo-mvc` to use the latest version of widgets/stores/cli.

Also update `todo-mvc-kitchensink` to use `dojo test`

### CODE REVIEW

File: todo-mvc-kitchensink/tests/functional/Page.ts [hunk 1]
```diff
@@ -76,7 +74,7 @@ export default class Page {
 
 	init(): Promise<any> {
 		return this.remote
-			.get(require.toUrl('../../src/index.html'))
+			.get('http://localhost:9000/_build/src/index.html')
```
Comment (dylans) [inline todo-mvc-kitchensink/tests/functional/Page.ts]: Do we want to hardcode a port/url, or read that from the Intern config?

Comment (agubler) [inline todo-mvc-kitchensink/tests/functional/Page.ts]: This should be temporary until changes are made for intern, I guess we could read this from the intern configuration... I really only mirrored what we did for the todo-mvc app.

Comment (agubler) [inline todo-mvc-kitchensink/tests/functional/Page.ts]: I'll address this in a follow up PR if you want me to.

Comment (dylans) [inline todo-mvc-kitchensink/tests/functional/Page.ts]: Sure, whenever you have time, it's not urgent.

**Pull request merged**